packages/harpguru-core/src/components/notify-of-quiz-question/utils/reduce-quizanswer-to-harpstrata/reduce-quizanswer-to-harpstrata.test.ts

```diff
@@ -34,10 +34,10 @@ test('when some toggles are buffered a new harpstrata is returned with those tog
   const activeHarpStrata = inactiveCellsHarpStrata
   const { Root: degreeQuizQuestion } = DegreeIds
   const { rootPitchId: pitchQuizQuestion } = activeHarpStrata
-  const bufferedActivityToggles = [DegreeIds.Second] as ReadonlyArray<DegreeIds>
+  const bufferedActivityToggles = [DegreeIds.Second] as const
   const expectedNewHarpStrata = {
     ...activeHarpStrata,
-    activeDegreeIds: [degreeQuizQuestion, DegreeIds.Second],
+    activeDegreeIds: [DegreeIds.Second, degreeQuizQuestion],
```

Comment:
Since the activeIds array can be left out of order by this action, I need to make sure that there is no way that the `getActiveIdsPair` function can end up mapping one way or the other inaccurately.

Reply:
Done. I don't think this was strictly necessary, but it's put my mind at ease.